reference/orders.v3.yml

```diff
@@ -2374,6 +2418,136 @@ components:
         - items
         - payments
       x-examples: {}
+    RefundID_Get:
+      type: object
+      properties:
+        data:
+          type: object
+          properties:
+            refund_id:
+              type: integer
+              description: |
+                Refund_ID for the return
+            order_id:
+              type: integer
+              description: |
+               Reference to order id
+            user_id:
+              type: integer
+              description: |
+                Reference to the user'd ID who created the refund. This is automatically populated by BigCommerce.
+            created:
+              type: string
+              description: |
+                Timestamp of when the refund was created
+            reason:
+              type: string
+              description: |
+                Reason for refund
+            total_amount:
+              type: number
+              description: |
+               A non-negative 2 decimal place rounded value that represents the amount that can be charged/refunded via payment providers.
```

Comment:
is `charged/refunded` correct? or should it be only `refunded` ? @bc-vincent-zhao